xml/System/Char.xml

```diff
@@ -1157,9 +1157,17 @@ When a managed <xref:System.Char> type, which is represented as a Unicode UTF-16
       </Parameters>
       <Docs>
         <param name="c">To be added.</param>
-        <summary>To be added.</summary>
+        <summary>Returns <see langword="true" /> if <paramref name="c" /> is an ASCII character ([ U+0000..U+007F ]).</summary>
         <returns>To be added.</returns>
```

Comment:
```suggestion
        <returns><see langword="true" /> if <paramref name="c"/> is an ASCII character; <see langword="false" /> otherwise.</returns>
```